autopilot/autopilot.py

```diff
@@ -2,11 +2,12 @@
 
 from bech32 import bech32_decode, CHARSET, convertbits
 from lib_autopilot import Autopilot, Strategy
-from lightning import LightningRpc, Plugin
+from lightning import LightningRpc, Plugin, RpcError
 import random
 import math
 import networkx as nx
 import dns.resolver
+import time
```

Comment:
`time` seems to be unused in the diff?

Reply:
Time is used to add a delay when peering with nodes (currently on line 50). The call is not in this diff because it was there originally, and the necessary import was missing.